docs/database/mirrored-database/open-mirroring-landing-zone-format.md

```diff
@@ -25,9 +25,9 @@ For every mirrored database, there is a unique storage location in OneLake for m
 
 For example, if you have tables (`Table A`, `Table B`, `Table C`) to be created in the landing zone, create folders like the following URLs:
 
-- `https://onelake.dfs.fabric.microsoft.com/<workspace id>/<mirrored database id>/LandingZone/TableA`
-- `https://onelake.dfs.fabric.microsoft.com/<workspace id>/<mirrored database id>/LandingZone/TableB`
-- `https://onelake.dfs.fabric.microsoft.com/<workspace id>/<mirrored database id>/LandingZone/TableC`
+- `https://onelake.dfs.fabric.microsoft.com/<workspace id>/<mirrored database id>/LandingZone/Files/TableA`
```

Comment:
Thanks for spotting the error and creating a PR for a fix. The right path should be `https://onelake.dfs.fabric.microsoft.com/<workspace id>/<mirrored database id>/Files/LandingZone/TableA`